examples/simple/posix/UB-PTHREAD6-bad.output

```diff
@@ -0,0 +1,7 @@
+Error: UB-PTHREAD6
+Description: Attempting to unlock a PTHREAD_MUTEX_DEFAULT or PTHREAD_MUTEX_NORMAL mutex which is not robust and which has not been locked by the current thread.
```

Comment:
It is considered undefined behavior, yes. Specifically it is considered undefined by the POSIX 2008 standard to unlock a mutex you do not own and have locked, unless the mutex is a  robust mutex, in which case it is well defined and returns a specific error code.